seqr/views/react_app_2_3_tests.py

```diff
@@ -13,8 +15,9 @@ class DashboardPageTest(AuthenticationTestCase):
     fixtures = ['users']
 
     def get_initial_page_json(self, response):
-        self.assertRegex(response.content, INITIAL_JSON_REGEX)
-        m = re.search(INITIAL_JSON_REGEX, response.content)
+        content = response.content.decode('utf-8')
+        self.assertRegex(content, INITIAL_JSON_REGEX)
+        m = re.search(INITIAL_JSON_REGEX, content)
```

Comment:
Python 3 re doesn't accept a `bytes` argument and the `response.content` is a `bytes`. So, decode the content to `str`.